chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/ActivityConceptEntity.java

```diff
@@ -13,18 +13,18 @@
 import gov.healthit.chpl.util.Util;
 
 @Entity
-@Table(name = "activity_class")
+@Table(name = "activity_concept")
```

Comment:
Does there need to be a data-model change too?

Reply:
Nope. That table doesn't exist. That entity wasn't referenced from anywhere which is why it never mattered but it happened to have the same class name that I wanted to use.

Reply:
Hah! Okay. Works for me :)